src/components/field/MetadataSync.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react'
+import i18n from '@dhis2/d2-i18n'
+import PropTypes from '@dhis2/prop-types'
+import { MANUAL, SyncSelect } from './SyncSelect'
+
+const CODE = 'metadataSync'
+const metadataOptions = [
+    {
+        value: '24h',
```

Comment:
Line 22 has a constant defined with this value. Does it make sense to use the constant here instead?